srcmedia/js/modules/Utilities.js

```diff
@@ -0,0 +1,19 @@
+export const toArray = value => Array.isArray(value) ? value : value ? [value] : undefined
+
+export const ajax = {
+    headers: { // this header is needed to signal an ajax request to Django
+        'X-Requested-With': 'XMLHttpRequest',
+    }
+}
+export const parser = new DOMParser()
+
+export const reduceFacets = (acc, cur) => {
```

Comment:
oh, these are from old implementations of the reactive search - good catch. Not used yet, but I put in since we may or may not need them when we start implementing the more complex text facet-type components.